### PR TITLE
Add perf check for inductor smoke test

### PR DESCRIPTION
**Pull request by weiwangmeta**
Background: performance smoke test job has been setup for inductor https://github.com/pytorch/pytorch/actions/workflows/inductor-perf-smoke-test.yml 
I have used this job to identify that https://github.com/pytorch/pytorch/pull/91254 regressed performance from 1.194x to 1.156x. However, this was by manual checking. 

To automatically flag similar regressions, we will add a reference value (which needs to be actively maintained) so that any speedups falling below the reference would be treated regression. 
 
In the back, two A100 instances from GCP would be running the perf check jobs for every push to upstream. So far these two instances give up to 1.204x and 1.197x, so we interpret any output below 1.185x to be suspicious.  

cc @mlazos @soumith @voznesenskym @yanboliang @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @chunyuan-w @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @desertfire

### CODE REVIEW

**Review by ngimel — APPROVED** [review]

File: .jenkins/pytorch/test.sh [hunk 1]
```diff
@@ -306,6 +306,9 @@ test_inductor_benchmark_perf() {
     python benchmarks/dynamo/torchbench.py --performance --backend inductor --float16 --training \
       --batch-size-file "$(realpath benchmarks/dynamo/torchbench_models_list.txt)" --only hf_Bert \
       --output "$TEST_REPORTS_DIR"/inductor_training_$1.csv
+    # the reference speedup value is hardcoded in check_hf_bert_perf_csv.py
+    # this value needs to be actively maintained to make this check useful
+    python check_hf_bert_perf_csv.py -f inductor_training_$1.csv
```
Comment (desertfire) [inline .jenkins/pytorch/test.sh:311]: Do you need to add `"$TEST_REPORTS_DIR"` before the csv filename?

Comment (weiwangmeta) [reply .jenkins/pytorch/test.sh:311]: Good catch, thank you! Yes, I will need to fix it.

Comment (weiwangmeta) [reply .jenkins/pytorch/test.sh]: Done.

**Review by desertfire — APPROVED** [review]

**Pull request closed without merging**